Remotion/ObjectBinding/Web/UI/Controls/Styles.cs

```diff
@@ -480,6 +485,20 @@ public override void CopyFrom (Style s)
       set { _rows = value; }
     }
 
+    ///  <summary>
+    /// Defines the autocomplete behaviour of the textbox.
+    /// Standard values are given in <see cref="BocTextValueAutocompleteAttribute"/>
+    /// </summary>
+    [Description("The autocomplete behaviour of the textbox.")]
+    [Category("Behavior")]
+    [DefaultValue(typeof(string), "")]
+    [NotifyParentProperty(true)]
+    public string? AutoComplete
```

Comment:
If I remember correct we decided that we should not use a nullable string here? Since we don't render the string in the empty case, we can just make this property not null and set the default value of the field to string.empty.